File: app/src/main/java/com/simplecity/amp_library/ui/drawer/DrawerView.java
package com.simplecity.amp_library.ui.drawer;

import com.simplecity.amp_library.model.Playlist;
import java.util.List;

public interface DrawerView {

    void setPlaylistItems(List<Playlist> playlists);

    void closeDrawer();

    void setDrawerItemSelected(@DrawerParent.Type int type);
}

File: app/src/main/java/com/simplecity/amp_library/ui/screens/drawer/DrawerView.kt
package com.simplecity.amp_library.ui.screens.drawer

import com.simplecity.amp_library.model.Playlist
import com.simplecity.amp_library.ui.screens.playlist.menu.PlaylistMenuContract
import com.simplecity.amp_library.ui.views.PurchaseView

interface DrawerView :
    PurchaseView,
    PlaylistMenuContract.View {

    fun setPlaylistItems(playlists: List<Playlist>)

    fun closeDrawer()

    fun setDrawerItemSelected(@DrawerParent.Type type: Int)
}